### ISSUE TITLE
Add unread indicator for direct messages

### ISSUE THREAD

**Issue by juliusknorr** [nextcloud/social#162]
Direct messages should have an indicator if they were already read, so we can properly show a count of unread messages in the navigation sidebar.

**Comment by jancborchardt** [comment]
Yup! For reference, documentation for the design of the counter is at https://docs.nextcloud.com/server/15/developer_manual/design/navigation.html#counter

**Comment by SchBenedikt** [comment]
Issue is solved with #2279
